docs/esp32/quickref.rst

```diff
@@ -365,6 +365,41 @@ Notes:
     
     p1 = Pin(4, Pin.OUT, None)
 
+RMT
+---
+
+The RMT (Remote Control) module, specific to the ESP32, was originally designed
+to send and receive infrared remote control signals. However, due to a flexible
+design and very accurate (as low as 12.5ns) pulse generation, it can also be
+used to transmit or receive many other types of digital signals::
```

Comment:
This kind of historical language is better suited to the reference docs of `esp32.RMT` rather than a quick ref.  For here I'd suggest something short and to the point like "The RMT is esp32-specific and allows generation of accurate digital pulse trains with 12.5ns resolution.  Usage is:".